src/components/card/TemplateCard.vue

```diff
@@ -95,8 +121,13 @@ export default {
     name: String,
     description: String,
     date: [Number, String],
+    type: [Number, String],
   },
   methods: {
+    getClass(className) {
```

Comment:
You are always calling this method with `type prop`. Create a computed property instead